src/App.js

```diff
@@ -1,16 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+  const [likeCount, setLikeCount] = useState(0);
+  
+  const updateLike = (id) => {
+    setMessages(messages => {
+      return messages.map(message => {
+        if (id === message.id) {
+          return {
+            ...message,
```

Comment:
Good use of object spread to update the relevant key/value while retaining the original key/value pairs!